### ISSUE TITLE
Who is the actual RP?

### ISSUE THREAD

**Issue by cyberphone** [rsolomakhin/secure-payment-confirmation#35]
In the original 3DS setup the ACS is used to assure the Merchant/Card network that the card -holder is authentic.  As far as I recall, the result of this authentication is _not_ brought back to the Issuer during the actual transaction request since that would involve changes in the core payment protocol as well as require interaction with the ACS.

Because the current SPC flow diagrams are pretty hard to follow, I'm unable to tell if this part has changed or not.

If this has not changed, I wouldn't characterize the Issuer as the RP.

**Comment by ianbjacobs** [comment]
I believe the issuer is the RP in the 3DS use case. Other parties will be RPs in other use cases (e.g., SRC). Closing for now but we can reopen if needed.
